mods/ts/rules/nod-support.yaml

```diff
@@ -41,6 +41,116 @@ NAGATE_B:
 	Tooltip:
 		Name: Nod Gate
 
+NAPOST:
+	Inherits: ^Building
+	Buildable:
+		Queue: Defense
+		BuildPaletteOrder: 150
+		Prerequisites: naapwr, ~structures.nod
+		Description: Stops infantry and blocks enemy fire.\nCan NOT be crushed by tanks.
+	SoundOnDamageTransition:
+		DamagedSound:
```

Comment:
This is wrong, in the original they did have the same transition sound other buildings had.